### ISSUE TITLE
Incorrect CRTM AMSU-A Spectral Coeff file on hera

### ISSUE THREAD

**Issue by ADCollard** [JCSDA/spack-stack#1046]
This is a similar issue to that already addressed in [Issue 963](https://github.com/JCSDA/spack-stack/issues/963#issuecomment-1910939035)

The AMSU-A Metop-C spectral coefficient file needs special treatment as the one supplied with the default file name is not correct.   The details on what to do are in that issue.

The correct file has the following `cksum`:
```
1237040622 12196 amsua_metop-c.SpcCoeff.bin
```
This one is from orion at `/work/noaa/epic/role-epic/spack-stack/orion/spack-stack-1.6.0/envs/unified-env/install/intel/2022.0.2/crtm-fix-2.4.0.1_emc-ezbyzji/fix`

On Hera at `/scratch1/NCEPDEV/nems/role.epic/spack-stack/spack-stack-1.6.0/envs/unified-env-rocky8/install/intel/2021.5.0/crtm-fix-2.4.0.1_emc-bm46d3q/fix` (which is the path given by $CRTM_FIX when the appropriate modules are loaded as specified in the develop version of GSI) the `cksum` is:
```
1692416252 1252 amsua_metop-c.SpcCoeff.bin
```

This causes a failure when trying to run the develop version of GSI.

**Comment by ADCollard** [comment]
@DavidHuber-NOAA Could you take a look at this if you have time?

**Comment by RussTreadon-NOAA** [comment]
**Hercules check**

Log into Hercules.  Load GSI `modulefiles/gsi_hercules.intel` and `cd $CRTM_FIX`.  The correct `amsua_metop-c.SpcCoeff.bin` is installed on Hercules
```
hercules-login-4:/work/noaa/epic/role-epic/spack-stack/hercules/spack-stack-1.6.0/envs/unified-env/install/intel/2021.9.0/crtm-fix-2.4.0.1_emc-2os2hw2/fix$ cksum amsua_metop-c.SpcCoeff.bin
1237040622 12196 amsua_metop-c.SpcCoeff.bin
```

Thank you @ADCollard for reporting this problem.  This issue prompted me to check GSI regression tests.  The global_4denvar test sets `ta2tb=.false.,`  This should be changed to `ta2tb=.true.` since this is how operations runs the global GSI.

**Comment by DavidHuber-NOAA** [comment]
@AlexanderRichert-NOAA @climbfuji @ADCollard I verified this does need to be fixed.  The procedure outlined in #963 should work fine again here.  This also needs to be fixed on Jet Rocky 8 installations.

**Comment by climbfuji** [comment]
@RatkoVasic-NOAA Can you work with @DavidHuber-NOAA to fix this, please?

**Comment by DavidHuber-NOAA** [comment]
@RatkoVasic-NOAA Apologies for not bringing this up in the spack-stack meeting.  Is this something that can be addressed?

**Comment by RatkoVasic-NOAA** [comment]
@DavidHuber-NOAA does this mean we just have to copy amsua_metop-c.SpcCoeff.bin to all our installations? If yes, which one we know is correct? And I can use that one to populate to all environments/versions.

**Comment by DavidHuber-NOAA** [comment]
@RatkoVasic-NOAA This only applies to spack-stack v1.6.0 Rocky 8 installations on Hera and Jet and only needs to be applied to the unified-env-rocky8 environment (the gsi-addon-rocky8 uses the unified-env-rocky8 installation of crtm-fix).

A good copy of the fix file can be found here:
`/scratch1/NCEPDEV/nems/role.epic/spack-stack/spack-stack-1.5.1/envs/unified-env-rocky8/install/intel/2021.5.0/crtm-fix-2.4.0_emc-sh2qz24/fix/amsua_metop-c.SpcCoeff.bin`

**Comment by RatkoVasic-NOAA** [comment]
@DavidHuber-NOAA thanks! I'm going to update it.

**Comment by RatkoVasic-NOAA** [comment]
@DavidHuber-NOAA BTW, I checked other files as well. Beside amsua_metop-c.SpcCoeff.bin, there are two more files that differ:
atms_j2.SpcCoeff.bin
atms_n21.SpcCoeff.bin
Do you know if that is expected?

**Comment by DavidHuber-NOAA** [comment]
@RatkoVasic-NOAA I'm not certain on this.
@ADCollard Do you know if these changes are expected between crtm-fix v2.4.0 and v2.4.0.1?

**Comment by ADCollard** [comment]
@RatkoVasic-NOAA @DavidHuber-NOAA Thanks for doing this!   This whole sorry saga started with atms_n21!  Hopefully for 2.4.0.1 the cksums should be as follows:

```
andrew.collard@dlogin06:/apps/ops/para/libs/intel/19.1.3.304/crtm/2.4.0.1/fix> cksum atms_n21.SpcCoeff.bin
3709800805 1960 atms_n21.SpcCoeff.bin
andrew.collard@dlogin06:/apps/ops/para/libs/intel/19.1.3.304/crtm/2.4.0.1/fix> cksum atms_j2.SpcCoeff.bin
4188410122 1960 atms_j2.SpcCoeff.bin
```

**Comment by RatkoVasic-NOAA** [comment]
@ADCollard thanks, it is correct in spack-stack-1.6.0/crtm-fix-2.4.0.1

**Comment by DavidHuber-NOAA** [comment]
The checksums and sizes are both correct now:
```
1237040622 12196 /lfs4/HFIP/hfv3gfs/role.epic/spack-stack/spack-stack-1.6.0/envs/unified-env-rocky8/install/intel/2021.5.0/crtm-fix-2.4.0.1_emc-z2uz6da/fix/amsua_metop-c.SpcCoeff.bin

1237040622 12196 /scratch1/NCEPDEV/nems/role.epic/spack-stack/spack-stack-1.6.0/envs/unified-env-rocky8/install/intel/2021.5.0/crtm-fix-2.4.0.1_emc-bm46d3q/fix/amsua_metop-c.SpcCoeff.bin
```

I think this can now be closed.  Thanks @RatkoVasic-NOAA!

**Comment by climbfuji** [comment]
Thank you both!
